2.4/root/usr/bin/run-mongod-replication

```diff
@@ -68,6 +68,7 @@ ${CONTAINER_SCRIPTS_PATH}/replset_supervisor.sh "${1:-}" "$$" &
 # Run `unset_env_vars` and `mongod` in a subshell because
 # MONGODB_ADMIN_PASSWORD should still be defined when the trapped call to
 # `cleanup` references it.
+set -m
```

Comment:
@php-coder It is now more explained in code

    # Don't pass signals to background processes - to succesfully remove local `mongod`
    # from replset in `cleanup()` mongod has to be running (to get addresses of other members).
    # Background processes are killed in `cleanup`.